### PR TITLE
chore(api): restructure product IDs

### PR DESCRIPTION
**Pull request by roalonso**

### CODE REVIEW

**Review by PabloReszczynski — APPROVED** [review]
👍

File: topsort-api-v2.yml [hunk 1]
```diff
@@ -379,17 +375,20 @@ components:
...
+        - ids
       type: object
       properties:
-        id:
-          type: string
-          description: >
-            The marketplace's ID of the product which will participate in the auction.
-            These ID must match those in the catalog integration with Topsort.
-          example: p_SA0238
+        ids:
```
Comment (sk-) [inline topsort-api-v2.yml:383]: @roalonso Should we add `minItems`?

Comment (roalonso) [reply topsort-api-v2.yml:383]: Ahh yes, and `maxItems`. Or should we add that limit later? we never discussed how many product ID we want to receive at most per request.

Comment (sk-) [reply topsort-api-v2.yml:383]: lets say 10k, that's higher than what we are receiving now

**Pull request merged**